micrometer-core/src/main/java/io/micrometer/core/instrument/binder/db/MetricsDSLContext.java

```diff
@@ -136,4972 +116,678 @@ public DSLContext tags(Iterable<Tag> tags) {
     }
 
     @Override
-    public Schema map(Schema schema) {
-        return context.map(schema);
+    public <R extends Record> SelectWhereStep<R> selectFrom(Table<R> table) {
```

Comment:
Any specific reason this is not 
```suggestion
public <R extends Record> SelectWhereStep<R> selectFrom(TableLike<R> table) {
```
?

Reply:
Because we compile against jooq 3.14 and `TableLike` does not exist in that version.